### PR TITLE
add r packages to conda env

### LINKED ISSUE #40: Possible Considerations on moving R into conda environment for consistency
Due to conda being able to handle the R dependencies as well, I'd recommend adding the following to the existing environment file:

r-vioplot
r-corrplot
r-gmodels
r-matrixstats
r-lmperm
r-pwr
r-fnn
r-klar
r-dmwr
r-xgboost
r-ellipse
r-mclust
r-ca

Optional: add rstudio-desktop version that is more supported
conda install -c conda-forge rstudio-desktop compared to the rstudio-desktop version that is part of the default.

[Here's the link to the conda-forge version of rstudio-desktop.](https://anaconda.org/conda-forge/rstudio-desktop)

### PR DESCRIPTION
**Pull request by gregorywaynepower**
This PR is to reduce the amount of software a user would have to install and prevent dependency conflicts on a user's machine if they used conda to manage r environments [Issue 40 for Reference](https://github.com/gedeck/practical-statistics-for-data-scientists/issues/40).

I have decided to exclude the[ conda-forge rstudio-desktop package](https://anaconda.org/conda-forge/rstudio-desktop) until I get an answer from Posit (formerly RStudio) on what repository of RStudio they manage in the conda / Anaconda ecosystem.

Closes #40

### CODE REVIEW

File: README.md
Comment (gedeck) [inline README.md:159]: We should probably still have a list of the dependencies or point the user to the `environment.yml` file for that list.

**Review by gedeck — APPROVED** [review]
Consider adding the comment that dependencies are listed in the `environments.yml` file

File: README.md
Comment (gregorywaynepower) [reply README.md:195]: I've added a list of dependencies to the README.md file, along with adding comments to the shell script.

**Comment by gedeck** [comment]
Are you able to merge the PR? If not, I will merge it.

**Comment by gregorywaynepower** [comment]
Since I don't have write access to your repo I can't merge it.

**Pull request merged**
